portfolio/src/main/webapp/script.js

```diff
@@ -150,3 +166,14 @@ function toggleImgs(element, containerName) {
     $(element).addClass("fa-sort-up").removeClass("fa-sort-down") : 
     $(element).addClass("fa-sort-down").removeClass("fa-sort-up");
 }
+
+/**
+ * Updates the message display based on query parameters if already displayed
+ */
+function correctDisplay() {
+  //only relevant if messages are already showing
+  if ($("#get-msg-btn").hasClass("invisible")) {
```

Comment:
Should this be "visible"?

Reply:
yes! that is what I meant but yea I see if could be confusing. Is there another way I could express this to make it more clear?

Reply:
I mean ideally you should have a class that maintain an internal boolean state `showComment` and use that to decide if a particular element is shown/hidden. It is pretty natural if you are using frameworks like react state and let the framework to rerender the components for you :)

But I admit that will be a lot of work and I don't think it worths it.